backend/controllers/prisonerProfile/adjudicationHistory.ts

```diff
@@ -54,9 +54,11 @@ export default ({
       systemOauthClient,
       restrictedPatientApi,
     })
+    const systemContext = await systemOauthClient.getClientCredentialsTokens(req.session.userDetails)
```

Comment:
Make calls to AdjudicationsAPI with system client credentials (`prisonstaffhubclient`) rather than the service credentials (`prison-staff-hub`) because those endpoints require `VIEW_ADJUDICATIONS` which is a service role, not a user one